uncontrast_studies/parsers/finding_parser.py

```diff
@@ -6,7 +6,7 @@
 
 
 def resolve_uncon_findings(item: dict, index: str):
-    NULL_VALUES = ["", "NA", "N/A", "n/a", "missing"]
+    NULL_VALUES = ["", "NA", "N/A", "n/a", "missing", "NaN", "nan"]
```

Comment:
seems to be a repeating list - consider extracting it to a common

Reply:
my end goal is to delete this anyway when the data is complete, but for now I can extract it